File: 프로그래머스/lv0/120882. 등수 매기기/등수 매기기.js
function solution(score) {
    const answer = score.map(([eng,math])=>{
        let count=1;
        for(let i=0; i<score.length; i+=1){
            if(eng+math<score[i][0]+score[i][1]) count+=1;
        }
        return count;
    })
    return answer;
}

File: 프로그래머스/lv0/120882. 등수 매기기/등수 매기기.py
def solution(score):
    answer = []
    scoremap = list(map(lambda x:[x,1],score))
    for i in scoremap:
        for j in scoremap:
            if i[0][0]+i[0][1] < j[0][0]+j[0][1]:
                i[1]+=1
    answer = list(map(lambda x:x[1],scoremap))
    return answer